frontend/server/src/Controllers/Course.php

```diff
@@ -111,9 +111,11 @@ private static function validateCreateAssignment(
             !is_null($finishTime) &&
             $startTime->time > $finishTime->time
         ) {
-            throw new \OmegaUp\Exceptions\InvalidParameterException(
+            $exception = new \OmegaUp\Exceptions\InvalidParameterException(
                 'courseInvalidStartTime'
             );
+            $exception->addCustomMessageToArray('errorcolumn', 'finish_time');
```

Comment:
dado que esta excepción se llama `InvalidParameterException`, se puede cambiar a

```suggestion
            $exception->addCustomMessageToArray('parameter', 'finish_time');
```

? Alternativamente, como `InvalidParameterException` ya espera que se le pase el nombre del parámetro como ... parámetro (je), no sería mejor cambiar https://github.com/omegaup/omegaup/blob/74de584df318ab7734fd22593294d7eeabe4a848/frontend/server/src/Exceptions/InvalidParameterException.php#L20-L28 para que lea

```php
    public function __construct(
        string $message,
        ?string $parameter = null,
        array $additionalParameters = []
    ) {
        parent::__construct($message, 'HTTP/1.1 400 BAD REQUEST', 400);
        $this->parameter = $parameter;
        $this->additionalParameters = $additionalParameters;
        if (!is_null($this->parameter)) {
            $exception->addCustomMessageToArray('parameter', 'finish_time');
        }
    }
```

y así este cambio se puede hacer significativamente más pequeño Y se pueden seguir arrojando excepciones idiomáticamente con `throw new ...` **Y** no nos tenemos que preocupar por andar agregando el parámetro a mano?

Reply:
Listo, sólo hice un cambio: no agregué la condición si `$this->parameter` es nulo y mejor envié el segundo parámetro en todos los lugares que se requiere para este formulario.